docs/example_code/quickstart.py

```diff
@@ -88,15 +104,20 @@
 print(f"media_object upload status: {upload_results.media_objects.status.value}")
 print(f"media object upload summary\n  {upload_results.media_objects.summary}")
 
+print(f"attribute upload status: {upload_results.attribute_objects.status.value}")
+print(f"attribute upload summary\n  {upload_results.attribute_objects.summary}")
```

Comment:
attribute_objects? they don't exist :)
```suggestion
print(f"attribute upload status: {upload_results.attributes.status.value}")
print(f"attribute upload summary\n  {upload_results.attributes.summary}")
```